src/org/labkey/test/components/react/DropdownButtonGroup.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2018-2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
```

Comment:
Remove copyright. This is no longer in a closed-source repository.

Reply:
thanks